buildSrc/src/main/kotlin/convention.publishing.gradle.kts

```
@@ -1,31 +1,33 @@
 import com.redmadrobot.build.dsl.*
+import com.vanniktech.maven.publish.SonatypeHost
 
 plugins {
-    id("com.redmadrobot.publish-config")
-    id("com.redmadrobot.publish")
+    id("com.vanniktech.maven.publish")
 }
 
-redmadrobot {
-    publishing {
-        signArtifacts = !isRunningOnCi
-        pom {
-            setGitHubProject("RedMadRobot/gears-android")
+mavenPublishing {
+    publishToMavenCentral(SonatypeHost.CENTRAL_PORTAL)
```

We should enable automatic release If we want the libraries to be released after `./gradlew publish`
```suggestion
    publishToMavenCentral(SonatypeHost.CENTRAL_PORTAL, automaticRelease = true)
```